CHANGES-5.0.md

```diff
@@ -1,3 +1,9 @@
+# 5.0.10
+
+## Enhancements
+
+* Use milliseconds internally in emqx_delayed to store the publish time, improving precision.[#9060](https://github.com/emqx/emqx/pull/9060)
```

Comment:
Should we put it in 5.0.9? it hasn't been released yet.

Reply:
OK, no problem